packages/auth/src/AuthService.ts

```diff
@@ -0,0 +1,63 @@
+import Keycloak from "keycloak-js";
+import {KeycloakError, KeycloakInitOptions, KeycloakInstance, KeycloakProfile, KeycloakPromise} from "keycloak-js";
+import { Key } from "readline";
+
+/**
+ * Wrapper class for {Keycloak.KeycloakInstance}
+ */
+export class AuthService {
+
+    private auth: KeycloakInstance;
+
+    constructor(config: KeycloakInitOptions) {
+      this.auth = Keycloak(config);
```

Comment:
Are you sure this doesn't need `new`? It's possible if the lib is a bit weird.

Reply:
@paolobueno It is actually needing new, but typescript types do not allow that.